.github/workflows/development-client-e2e.yml

```
@@ -48,3 +48,9 @@ jobs:
         run: |
           yarn e2e
         working-directory: packages/expo-dev-client
+      - name: Store artifacts of build failures
+        if: always()
```

I guess you wanted to run it on failures 😄
```suggestion
        if: failure()
```